pkg/eosclient/eosgrpc/eosgrpc.go

```diff
@@ -99,12 +90,11 @@ type Options struct {
 	// SecProtocol is the comma separated list of security protocols used by xrootd.
 	// For example: "sss, unix"
 	SecProtocol string
+
+	httpopts ehttp.Options
```

Comment:
Capitalize the first letter

Reply:
I can, yet that was meant to be a private member. Do you think I have to make it public?